tools/scripts/data.toml

```diff
@@ -166,72 +114,3 @@ category = "ICU4X Data"
 dependencies = [
     "testdata-check-unchanged",
 ]
-
-[tasks.testdata-build-bincode]
-description = "Build ICU4X Bincode filesystem structure from the downloaded CLDR JSON, overwriting the existing ICU4X Bincode if present."
-category = "ICU4X Data"
-command = "cargo"
-args = [
-    "run",
-    "--bin=icu4x-datagen",
-    "--features=experimental,bin",
-    "--",
-    "--input-from-testdata",
-    "--out-testdata",
-    "--test-keys",
-    "--test-locales",
-    "--syntax=bincode",
-    "--overwrite",
-]
-
-[tasks.testdata-build-postcard-tree]
```

Comment:
Issue: Many of the tasks down here are for debugging or CI purposes. I can tell you what they're for:

- `testdata-build-bincode` ensures that our bincode generation code is covered. We used to run "testdata-build-bincode-all", and since https://github.com/unicode-org/icu4x/commit/21978768b9009be591481a614cc0f8c1a524acef now we run "testdata-build-worklog-bincode" for that purpose.
- `testdata-build-postcard-tree` builds individual postcard files for testdata, which I find useful for debugging when postcard files unexpectedly increase in size.
- `testdata-build-bincode-all` runs Bincode generation on all locales, which is greater than testdata locales, because there are some locales that are in the CLDR source data directory that are not in the testdata locales list.
- `testdata-build-worklog-bincode` covers the `--key-file` option.

Reply:
Ok so:
* `testdata-build-bincode`: `testdata-build-hello-world` now builds the hello world data in all three serde formats, and stores them in `provider/{fs,blob}/tests/data`, where they are covered by tests. This gives us coverage of the whole pipeline for all three formats without depending on actual data. I think this is the reason why the hello world key exists in the first place.
* `testdata-build-postcard-tree` we have the hashes now, so I'm not convinced we still need this command.
* `testdata-build-bincode-all`: Bincode is covered by hello world, and I've added a `full-data` task that uses the latest CLDR and exportdata. We should add this to CI, but it's currently failing because the `ucase.toml` file has disappeared from the export data again (?!)
* `testdata-build-worklog-bincode` is now covered by a unit test for the `keys_from_file` function

Reply:
Ok, but please restore `testdata-build-postcard-tree` in some form